src/RileyMaze/userFunction.js

```diff
@@ -198,7 +186,10 @@ ${code}
         const {name, args, getScope} = USER_FUNCTIONS[this._codeview.userFunction];
         const code = this._codeview.text;
         const scope = getScope();
-        scope.riley.queue.length = 0;
+
+        if (scope.hasOwnProperty('riley'))
```

Comment:
This seems like it should be moved into the Instructions topic's `getScope` function, since the `riley` object only applies there. I'd like to avoid special-casing things in the UserFunction class because I think the common user function code should eventually be factored out of Lightspeed and Sidetrack.